src/actions/index.ts

```diff
@@ -18,6 +18,11 @@ export const updateWalletBalance = (ethBalance: any, hatsBalance: any) => ({
   hatsBalance
 })
 
+export const updateNetwork = (network: string) => ({
```

Comment:
It's necessary to have the current `NETWORK` in the redux? instead of just checking it using `getNetworkNameByChainId`

Reply:
misunderstanding